test/acceptance/ping.controller.acceptance.ts

```diff
@@ -5,7 +5,7 @@
 
 import {createClientForHandler, supertest} from '@loopback/testlab';
 import {RestServer} from '@loopback/rest';
-import {ShoppingApplication} from '../';
+import {ShoppingApplication} from '../..';
```

Comment:
@shimks @virkt25 could you please create a follow up issue or open a pull request to fix the template? Otherwise we are most likely going to forget about this.